test/fluentd-forward.sh

```diff
@@ -33,63 +32,75 @@ update_current_fluentd() {
     # update configmap secure-forward#.conf
     if [ $cnt -eq 1 ]; then
       # edit so we don't send to ES
-      oc get configmap/logging-fluentd -o yaml | sed '/## matches/ a\
+      oc get configmap/logging-fluentd -o yaml | sed -e '/## matches/ a\
```

Comment:
Do we need '-e'???

Reply:
not strictly, but will save some confusion if another sed expression is added later

Reply:
sorry - yes - it is needed because there are more than 1 expression for this sed now

Reply:
ok.